packages/editor/src/components/post-text-editor/test/index.js

```diff
@@ -175,4 +176,28 @@ describe( 'PostTextEditor', () => {
 
 		expect( textarea.props.value ).toBe( 'Goodbye World' );
 	} );
+	it( 'throttle value update after given time', () => {
+		let wrapper;
+		act( () => {
+			wrapper = create( <PostTextEditor /> );
+		} );
+		const mockDispatchFn = jest.fn();
+		const useDispatchSpy = new Proxy( wp, {
```

Comment:
Why use a `Proxy` here and import all `@wordpress/data`?

You could just mock `useDispatch` with jest like this:
```
jest.mock( '@wordpress/data/src/components/use-dispatch', () => ( {
	useDispatch: () => ( { editPost: jest.fn(), ....etc } ),
} ) );
```